### PR TITLE
Enhance: disallow duplicate entries when upsert svc-disc config & other fixes

### PR DESCRIPTION
**Pull request by piglei**
- Disallow duplicate entries when upsert svc-disc config
- Changed the behavior of `SvcDiscEntryBkSaaSSLZ`: it sets `module_name` to None instead of `default', because `None` means that the **default module** is always used, while `default' refers to the module named "default" even if it is not the real "default" module.
- Change the permission level for APIs: DomainResolutionViewSet / SvcDiscConfigViewSet
- Some translation issues fixed

### CODE REVIEW

File: apiserver/paasng/paasng/platform/bkapp_model/serializers.py [hunk 1]
```diff
@@ -106,23 +109,37 @@ class SvcDiscEntryBkSaaSSLZ(serializers.Serializer):
...
+
 class SvcDiscConfigSLZ(serializers.Serializer):
     bk_saas = serializers.ListField(help_text="服务发现列表", child=SvcDiscEntryBkSaaSSLZ())
 
+    def validate_bk_saas(self, value: BkSaaSList) -> BkSaaSList:
+        seen = set()
+        for item in value:
+            pair = (item["bkAppCode"], item["moduleName"] or "")
+            if pair in seen:
+                raise serializers.ValidationError(_("服务发现列表中存在重复项：{}").format(pair))
```
Comment (shabbywu) [inline apiserver/paasng/paasng/platform/bkapp_model/serializers.py:139]: 抛到前端的文案会有点怪。

File: apiserver/paasng/paasng/platform/bkapp_model/serializers.py [hunk 2]
```diff
@@ -106,23 +109,37 @@ class SvcDiscEntryBkSaaSSLZ(serializers.Serializer):
     """A service discovery entry that represents an application and an optional module."""
 
     bk_app_code = serializers.CharField(help_text="被服务发现的应用 code", max_length=20, source="bkAppCode")
-    module_name = serializers.CharField(
-        help_text="被服务发现的应用模块", max_length=20, default=ModuleName.DEFAULT.value, source="moduleName"
-    )
+    module_name = serializers.CharField(help_text="被服务发现的模块", max_length=20, source="moduleName", default=None)
```
Comment (shabbywu) [inline apiserver/paasng/paasng/platform/bkapp_model/serializers.py:112]: 现在 moduleName = None 时，拿的是 name="default" 的模块还是 is_default=True 的模块？

**Review by shabbywu — APPROVED** [review]

File: apiserver/paasng/paasng/platform/bkapp_model/serializers.py (hunk 2 above)
Comment (piglei) [reply apiserver/paasng/paasng/platform/bkapp_model/serializers.py:112]: 这个：`is_default=True 的模块 `

File: apiserver/paasng/paasng/platform/bkapp_model/serializers.py (hunk 1 above)
Comment (piglei) [reply apiserver/paasng/paasng/platform/bkapp_model/serializers.py:139]: 是的，写的时候“偷懒”了，没有去做条件拼接。我改一下。
```
服务发现列表中存在重复项：[AppID: foo]
服务发现列表中存在重复项：[AppID: foo, ModuleName: bar]
```

**Review by SheepSheepChen — APPROVED** [review]

**Pull request merged**